[2018-11-24 14:05:44] php.DEBUG: Warning: filemtime(): stat failed for /var/www/mescyt/insys3/app/config/routing_dev.yml {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/var/www/mescyt/insys/vendor/symfony/symfony/src/Symfony/Component/Config/Resource/FileResource.php\",\"line\":63,\"trace\":[{\"file\":\"/var/www/mescyt/insys/vendor/symfony/symfony/src/Symfony/Component/Config/Resource/SelfCheckingResourceChecker.php\",\"line\":34,\"function\":\"isFresh\",\"class\":\"Symfony\\\\Component\\\\Config\\\\Resource\\\\FileResource\",\"type\":\"->\"}],\"count\":1})"} []
[2018-11-24 14:05:44] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_class" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_class\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerMfxbqqp/getSensioFrameworkExtra_Routing_Loader_AnnotClassService.php:8)"} []
[2018-11-24 14:05:44] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Routing\AnnotatedRouteControllerLoader" class is deprecated since version 5.2. Use "Symfony\Bundle\FrameworkBundle\Routing\AnnotatedRouteControllerLoader" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Routing\\AnnotatedRouteControllerLoader\" class is deprecated since version 5.2. Use \"Symfony\\Bundle\\FrameworkBundle\\Routing\\AnnotatedRouteControllerLoader\" instead. at /var/www/mescyt/insys/vendor/sensio/framework-extra-bundle/Routing/AnnotatedRouteControllerLoader.php:19)"} []
[2018-11-24 14:05:44] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_dir" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_dir\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerMfxbqqp/getSensioFrameworkExtra_Routing_Loader_AnnotDirService.php:8)"} []
[2018-11-24 14:05:44] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_file" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_file\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerMfxbqqp/getSensioFrameworkExtra_Routing_Loader_AnnotFileService.php:8)"} []
[2018-11-24 14:05:44] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:05:44] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:05:44] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2018-11-24 14:06:47] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:06:47] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:06:47] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2018-11-24 14:07:34] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:07:34] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:07:34] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:07:34] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:07:34] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:07:34] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'insys.usuario' doesn't exist" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 53 {"exception":"[object] (Doctrine\\DBAL\\Exception\\TableNotFoundException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S02]: Base table or view not found: 1146 Table 'insys.usuario' doesn't exist at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:53, Doctrine\\DBAL\\Driver\\PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'insys.usuario' doesn't exist at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'insys.usuario' doesn't exist at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:09:26] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:09:26] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:09:26] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:09:26] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:09:26] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:09:26] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:09:29] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:09:29] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:09:29] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:09:29] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:09:29] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:09:29] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:09:33] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:09:33] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:09:33] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:09:33] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:09:33] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:09:33] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:09:46] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:09:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:09:46] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:09:46] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:09:46] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:09:46] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:09:48] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:09:48] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:09:48] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:09:48] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:09:48] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:09:48] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:09:51] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:09:51] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:09:51] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:09:51] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:09:51] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:09:51] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:09:59] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:09:59] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:09:59] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:09:59] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:09:59] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:09:59] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:10:01] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:10:01] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:10:01] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:10:01] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:10:01] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:10:01] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:10:02] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"3c807e","_route":"_wdt"},"request_uri":"http://insys/app_dev.php/_wdt/3c807e","method":"GET"} []
[2018-11-24 14:10:03] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:10:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:10:03] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:10:03] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:10:03] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:10:03] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:10:12] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:10:12] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:10:12] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:10:12] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:10:12] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:10:12] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:19:01] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:19:01] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:19:01] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:19:01] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:19:01] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:19:01] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:19:42] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:19:42] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:19:42] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:19:42] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:19:42] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:19:42] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:20:47] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:20:47] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:20:47] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:20:47] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:20:47] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:20:47] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:20:48] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"f9a413","_route":"_wdt"},"request_uri":"http://insys/app_dev.php/_wdt/f9a413","method":"GET"} []
[2018-11-24 14:21:08] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:21:08] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:21:08] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:21:08] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:21:08] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:21:08] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 14:21:39] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:21:39] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:21:40] request.CRITICAL: Uncaught PHP Exception RuntimeException: "The autoloader expected class "AppBundle\Entity\Usuario" to be defined in file "/var/www/mescyt/insys/vendor/composer/../../src/AppBundle/Entity/Usuario.php". The file was found but the class was not in it, the class name or namespace probably has a typo." at /var/www/mescyt/insys/vendor/symfony/symfony/src/Symfony/Component/Debug/DebugClassLoader.php line 205 {"exception":"[object] (RuntimeException(code: 0): The autoloader expected class \"AppBundle\\Entity\\Usuario\" to be defined in file \"/var/www/mescyt/insys/vendor/composer/../../src/AppBundle/Entity/Usuario.php\". The file was found but the class was not in it, the class name or namespace probably has a typo. at /var/www/mescyt/insys/vendor/symfony/symfony/src/Symfony/Component/Debug/DebugClassLoader.php:205)"} []
[2018-11-24 14:21:40] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"faf6a2","_route":"_wdt"},"request_uri":"http://insys/app_dev.php/_wdt/faf6a2","method":"GET"} []
[2018-11-24 14:23:06] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:23:06] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:23:06] php.CRITICAL: Fatal Compile Error: Cannot declare class AppBundle\AppBundle, because the name is already in use {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: Cannot declare class AppBundle\\AppBundle, because the name is already in use at /var/www/mescyt/insys/src/AppBundle/Entity/Usuario.php:0)"} []
[2018-11-24 14:23:06] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: Cannot declare class AppBundle\AppBundle, because the name is already in use" at /var/www/mescyt/insys/src/AppBundle/Entity/Usuario.php line 0 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: Cannot declare class AppBundle\\AppBundle, because the name is already in use at /var/www/mescyt/insys/src/AppBundle/Entity/Usuario.php:0)"} []
[2018-11-24 14:23:07] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"49b98f","_route":"_wdt"},"request_uri":"http://insys/app_dev.php/_wdt/49b98f","method":"GET"} []
[2018-11-24 14:24:37] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:24:37] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:24:37] php.CRITICAL: Call to undefined method AppBundle\Entity\Usuario::setApellido() {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to undefined method AppBundle\\Entity\\Usuario::setApellido() at /var/www/mescyt/insys/src/AppBundle/Controller/DefaultController.php:19)"} []
[2018-11-24 14:24:37] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "setApellido" of class "AppBundle\Entity\Usuario"." at /var/www/mescyt/insys/src/AppBundle/Controller/DefaultController.php line 19 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"setApellido\" of class \"AppBundle\\Entity\\Usuario\". at /var/www/mescyt/insys/src/AppBundle/Controller/DefaultController.php:19)"} []
[2018-11-24 14:24:37] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"00a4c0","_route":"_wdt"},"request_uri":"http://insys/app_dev.php/_wdt/00a4c0","method":"GET"} []
[2018-11-24 14:26:40] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_class" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_class\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerStegfsk/getSensioFrameworkExtra_Routing_Loader_AnnotClassService.php:8)"} []
[2018-11-24 14:26:40] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Routing\AnnotatedRouteControllerLoader" class is deprecated since version 5.2. Use "Symfony\Bundle\FrameworkBundle\Routing\AnnotatedRouteControllerLoader" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Routing\\AnnotatedRouteControllerLoader\" class is deprecated since version 5.2. Use \"Symfony\\Bundle\\FrameworkBundle\\Routing\\AnnotatedRouteControllerLoader\" instead. at /var/www/mescyt/insys/vendor/sensio/framework-extra-bundle/Routing/AnnotatedRouteControllerLoader.php:19)"} []
[2018-11-24 14:26:40] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_dir" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_dir\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerStegfsk/getSensioFrameworkExtra_Routing_Loader_AnnotDirService.php:8)"} []
[2018-11-24 14:26:40] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_file" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_file\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerStegfsk/getSensioFrameworkExtra_Routing_Loader_AnnotFileService.php:8)"} []
[2018-11-24 14:26:40] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Configuration\Route" annotation is deprecated since version 5.2. Use "Symfony\Component\Routing\Annotation\Route" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Configuration\\Route\" annotation is deprecated since version 5.2. Use \"Symfony\\Component\\Routing\\Annotation\\Route\" instead. at /var/www/mescyt/insys/vendor/sensio/framework-extra-bundle/Configuration/Route.php:16)"} []
[2018-11-24 14:26:40] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:26:40] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:26:40] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"c1eb9f","_route":"_wdt"},"request_uri":"http://insys/app_dev.php/_wdt/c1eb9f","method":"GET"} []
[2018-11-24 14:29:39] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_class" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_class\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerStegfsk/getSensioFrameworkExtra_Routing_Loader_AnnotClassService.php:8)"} []
[2018-11-24 14:29:39] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Routing\AnnotatedRouteControllerLoader" class is deprecated since version 5.2. Use "Symfony\Bundle\FrameworkBundle\Routing\AnnotatedRouteControllerLoader" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Routing\\AnnotatedRouteControllerLoader\" class is deprecated since version 5.2. Use \"Symfony\\Bundle\\FrameworkBundle\\Routing\\AnnotatedRouteControllerLoader\" instead. at /var/www/mescyt/insys/vendor/sensio/framework-extra-bundle/Routing/AnnotatedRouteControllerLoader.php:19)"} []
[2018-11-24 14:29:39] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_dir" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_dir\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerStegfsk/getSensioFrameworkExtra_Routing_Loader_AnnotDirService.php:8)"} []
[2018-11-24 14:29:39] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_file" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_file\" service is deprecated since version 5.2 at /var/www/mescyt/insys/app/cache/dev/ContainerStegfsk/getSensioFrameworkExtra_Routing_Loader_AnnotFileService.php:8)"} []
[2018-11-24 14:29:39] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Configuration\Route" annotation is deprecated since version 5.2. Use "Symfony\Component\Routing\Annotation\Route" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Configuration\\Route\" annotation is deprecated since version 5.2. Use \"Symfony\\Component\\Routing\\Annotation\\Route\" instead. at /var/www/mescyt/insys/vendor/sensio/framework-extra-bundle/Configuration/Route.php:16)"} []
[2018-11-24 14:29:39] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 14:29:39] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 14:29:39] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 14:29:39] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 14:29:39] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 14:29:39] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
[2018-11-24 15:15:45] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://insys/app_dev.php/","method":"GET"} []
[2018-11-24 15:15:45] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-24 15:15:45] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-11-24 15:15:45] doctrine.DEBUG: INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?) {"1":"Juan","2":"juanp@gmail.com","3":"De Los Palotes","4":"1234","5":true} []
[2018-11-24 15:15:45] doctrine.DEBUG: "ROLLBACK" [] []
[2018-11-24 15:15:45] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params ["Juan", "juanp@gmail.com", "De Los Palotes", "1234", 1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list'" at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'INSERT INTO usuario (nombre, email, apellido, password, habilitado) VALUES (?, ?, ?, ?, ?)' with params [\"Juan\", \"juanp@gmail.com\", \"De Los Palotes\", \"1234\", 1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:107, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'password' in 'field list' at /var/www/mescyt/insys/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:105)"} []
